test/affprop.jl

```diff
@@ -6,6 +6,7 @@ using Clustering
 using LinearAlgebra
 using Random
 using Statistics
+include("test_helpers.jl")
```

Comment:
Do we like reincluding this in all the files? (rather than in runtests.jl)
It means we can run each file seperately, which I guess is the reason all the `using`s are repeated.
So I guess it is right to follow the pattern :-) ?